### ISSUE TITLE
data-cycle-hash values with multiple slideshows

### ISSUE THREAD

**Issue by matt-bernhardt** [malsup/cycle2#395]
Hello, I've been trying to get the following to work but am running into difficulties. I'm starting to wonder if I'm just asking too much:

I have four slideshows, which are each displayed inside a vertical accordion (slideshow A is in the first panel, slideshow B in the second panel, etc). Every slide of each slideshow should be individually shareable with its own URL.

I've been able to use data-cycle-hash to address slides within a single slideshow, and using a combination of BBQ and jQueryUI accordion I can address a single panel from an accordion. Getting these two systems to play together has been ... challenging. I'm working on a jsFiddle to replicate what I've done so far, but I wanted to check to see whether I'm missing something obvious.

The sticking point I'm at right now is that, when I include data-cycle-hash values for each slide, the last slideshow's value is the one that trumps anything else. The preferred behavior is for the default to be the first slide of the first slideshow.

**Comment by matt-bernhardt** [comment]
Okay, I have a jsFiddle set up that gets close to what I'm thinking - http://jsfiddle.net/matt_bernhardt/J6CAA/

**Comment by malsup** [comment]
For starters, the version of jQuery you're using isn't compatible with the version of BBQ you're using.  You've got an error occurring.

**Comment by matt-bernhardt** [comment]
Thanks for catching that - I've updated the fiddle accordingly. I'm also looking at a few other pieces of the BBQ documentation, and the Cycle2 API documentation. 

After thinking about this more this evening, I'm not sure that I have a technical question/problem - beyond the rather vague "am I crazy for trying to do this".

**Comment by matt-bernhardt** [comment]
I'm closing this - I got it working. The new fiddle is here: http://jsfiddle.net/matt_bernhardt/E7zay/ 

What I ended up doing (which may not be the best way, but it appears to work) is to not use the data-cycle-hash attributes. I couldn't get cycle2 to not overwrite other parameters that were present in the hash. Instead, I used BBQ to store the current slide in a "slide" key, while the accordion state was being stored in an "active" key. The nature of jsFiddle doesn't show it, but the URL is bookmarkable/shareable, and the state of both plugins is preserved.
